src/backend/distributed/planner/multi_router_planner.c

```diff
@@ -1043,3 +1055,24 @@ ColumnMatchExpressionAtTopLevelConjunction(Node *node, Var *column)
 
 	return false;
 }
+
```

Comment:
We typically leave 2 empty lines between functions. You can use the citus_indent tool to automatically enforce our code style.